postnl_api/postnl_api.py

```diff
@@ -7,8 +7,8 @@
 
 import requests
 
-from postnl_api.items.package import Package
-from postnl_api.items.letter import Letter
+from items.package import Package
```

Comment:
I am not sure about this change, need to test it myself... 
@eavanvalkenburg does this change matter? (/ is it a breaking change?)

Reply:
I had already issues in my original PR https://github.com/iMicknl/python-postnl-api/pull/10 but since @eavanvalkenburg continued with my PR I thought the issue was related to my environment. However, after rechecking again, I was unable to run the test case without this change and the fact that there was an error within the test case itself, probably means no-one ever ran it?

Reply:
@peternijssen, I can't run the separate test script without your changes indeed. However my linting throws errors after this change in the original script. Possibly we need to export it the right way, however I am not a Python guru.

@eavanvalkenburg could you have a look?

Reply:
@iMicknl I reverted this change for now, so you can still merge and release the rest. I performed a manual revert, so @eavanvalkenburg can still review the original change and comment if it's wrong or right.

Could you accept this PR and release a version? Thanks!

Reply:
I would create a new virtualenv to test it properly, I haven't had time to look yet, but might be able to this week.

Reply:
@iMicknl how do you wanna proceed? There are at least 2 obvious bugs that are fixed in the current PR.